src/pages/Validators/ValidatorsTable.tsx

```diff
@@ -321,9 +321,17 @@ type ValidatorRowProps = {
 function ValidatorRow({validator, columns}: ValidatorRowProps) {
   const inDev = useGetInDevMode();
   const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
 
   const rowClick = (address: Types.Address) => {
-    navigate(`/validator/${address}`);
+    // TODO(jill) find long term to persist the url params
+    const isUsingDev = searchParams.get("feature");
```

Comment:
there should be a hook somewhere `isDevMode` (or something like that), why not using it?